wp-content/themes/reconasia/functions.php

```diff
@@ -603,3 +603,10 @@ function reconasia_filter_excerpt ($post_excerpt) {
   return $post_excerpt;
 }
 add_filter ('get_the_excerpt','reconasia_filter_excerpt');
+
+
+/** Modify Tag List Classes */
+function reconasia_filter_post_tag_term_links( $links ) {
+	return str_replace('<a href="', '<a class="btn--xsmall" href="', $links);
```

Comment:
Is there a reason that `btn--xsmall` couldn't be applied to the `<li>` like the `btn--dark` class was? Feels odd to me to separate them.

Edit: Just saw, it's because the btn class overrides it. We should probably correct that in our button component so that doesn't bite us in the butt somewhere else. We can do that by modifying the selector like this:
```
&:not(.btn--xsmall):not(.btn--small):not(.btn--med),
&--large {
    @extend %text-style-label-large;
}
```